algebird-core/src/main/scala/com/twitter/algebird/BloomFilter.scala

```diff
@@ -225,6 +236,12 @@ case class BFInstance(hashes: BFHash, bits: BitSet, width: Int) extends BF {
       width)
   }
 
+  def checkAndAdd(item: String): (BF, ApproximateBoolean) = {
+    val bitsToActivate = BitSet(hashes(item): _*)
```

Comment:
Good catch, actually this was not complete - to re-use hashes we will need more changes - all in latest commit.